src/assets/scripts/client/App.js

```diff
@@ -50,8 +51,55 @@ export default class App {
         this.prop.log = LOG.DEBUG;
         this.prop.loaded = false;
 
-        return this.setupHandlers()
-            .loadInitialAirport(airportLoadList, initialAirportToLoad);
+        $.getJSON('assets/airports/airportLoadList.json')
+            .done((data) => {
+                // List of airports to load
+                const airportLoadList = data.filter((airport) => airport.disabled !== true);
+
+                // ICAO id of the initial airport. may be the default or a stored airport
```

Comment:
this is a good useful comment 👍

Reply:
Can't actually take credit for this, was also moved from JSDoc. :stuck_out_tongue_closed_eyes: